udata/core/spatial/commands.py

```diff
@@ -44,7 +44,10 @@ def load_levels(col, json_levels):
 
 
 def load_zones(col, json_geozones):
+    loaded_geozones = 0
     for i, geozone in enumerate(json_geozones):
```

Comment:
Here `i` is not a total, it is an index used as a total. This index is not relevant anymore if we do not load every zone returned by `enumerate`.

Reply:
Why do you keep the `i` if it is not used?